README

```diff
@@ -929,17 +929,18 @@ is simpler and more straightforward than moving around using the menu.
 --------------------------------------
 To properly plan the World of Xeen CD Talkie using original discs,
 use LAME or some other MP3 encoder to rip the cd audio tracks to files.
-Name the files track1.mp3 track2.mp3 etc. ScummVM must be compiled with
```

Comment:
I'm just curious, why this change?

Reply:
I thought it would avoid confusion, since my understanding is that on original discs, the first track is the data one, so the extracted tracks would start with track 2. So I wanted to avoid confusing the users into thinking they'd have to rename the 2nd track to track 1, 3rd to track 2, etc.

Reply:
The GOG track list also starts with xeen02.ogg, so keeping extracted tracks as default has both methods matching.

Reply:
Gotcha, thanks!